src/core/components/ChatView.tsx

```diff
@@ -87,7 +89,10 @@ export const ChatView = ({
           >
             {!hasMessages && <LandingContent />}
 
-            <ChatInput setShouldAutoScroll={setShouldAutoScroll} />
+            <ChatInput
+              setShouldAutoScroll={setShouldAutoScroll}
+              supportsUpload={supportsUpload}
+            />
```

Comment:
This component will be more easily reused if it can accept a prop to support uploads or not.